corehq/ex-submodules/casexml/apps/phone/models.py

```diff
@@ -750,6 +750,14 @@ class SimplifiedSyncLog(AbstractSyncLog):
     closed_cases = SetProperty(unicode)
     extensions_checked = BooleanProperty(default=False)
 
+    _purged_cases = None
+
+    @property
+    def purged_cases(self):
+        if not self._purged_cases:
```

Comment:
nitpick: could now change this to `if self._purged_cases is None:` so it does not continually create a new `set` each time `self.purged_cases` is accessed while the set is empty.